### DISCUSSION TITLE
Gist Links can not open on mobile app

### DISCUSSION THREAD

**Discussion by AhmetEnesKCC** [community/community#65194] [Mobile]
### Select Topic Area

Bug

### Body

Github Mobile App has not got the feature of opening gists. However when i click gist link from slack or another app it is trying to open with mobile application. I assume that this is a bug about github mobile app.

**Comment by arafajs** [comment]
You are correct. The GitHub Mobile App does not currently have the feature of opening gists. However, when you click a gist link from Slack or another app, it is trying to open with the mobile application. This is a bug in the GitHub Mobile App.

There have been several discussions on the GitHub Community forum about this issue, and GitHub is aware of the problem. They have not yet announced a fix for this issue, but they are working on it.

There are a few things you can do to workaround this issue:

1. Open the gist in a web browser.
2. Use a third-party app to open gists. There are a few apps available that can open gists, such as GistHub.
3. Wait for GitHub to fix the issue.

**Comment by 0dayhunter** [comment]
I apologize for any misunderstanding. Based on the information you provided, it does appear that there might be a bug in the GitHub Mobile App related to opening Gist links. When you click a Gist link from Slack or another app, the GitHub Mobile App attempts to open it, but it seems that the expected behavior is not being achieved.

Since I don't have access to real-time information about the GitHub Mobile App's current status or updates beyond September 2021, I recommend taking the following steps:

1. **Report the Bug:** If you haven't already, report this issue to GitHub. You can do this by reaching out to their support or using any feedback mechanisms provided in the app. This will help the development team become aware of the problem and potentially work on a solution.

2. **Check for Updates:** Regularly update the GitHub Mobile App. New versions often come with bug fixes and improvements. It's possible that this issue might have been addressed in a subsequent update.

3. **Community Discussions:** Look for GitHub's official community forums, social media accounts, or platforms where users discuss the app. You might find other users who are experiencing the same issue or have found workarounds.

4. **Alternate Access:** In the meantime, consider using a web browser on your mobile device to access Gists directly on the GitHub website.

5. **Patience:** Bugs can take some time to be fixed. Be patient and monitor for updates from GitHub.

To reiterate, I recommend reaching out to GitHub's official support channels or community platforms for the most accurate and up-to-date information regarding this issue and its potential resolution.
